packages/front-end/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,79 @@
+import clsx from "clsx";
+import { MdCheck } from "react-icons/md";
+import { PiCircleDashed } from "react-icons/pi";
+
+type Step = {
+  label: string;
+  enabled: boolean;
+};
+
+interface Props {
+  step: number;
+  setStep: (step: number) => void;
+  steps: Step[];
+  setError: (error: string) => void;
+  skipped?: Set<number>;
+  validateSteps?: (step: number) => Promise<void>;
+}
+
+export default function Stepper({
+  step,
+  setStep,
+  steps,
+  skipped,
+  setError,
+  validateSteps,
+}: Props) {
+  return (
+    <nav
+      className={
+        "nav mb-4 justify-content-start nav-default paged-modal-default"
+      }
+    >
+      {steps.map(({ label, enabled }, i) => {
+        return (
+          <div
+            className={clsx(
+              "step d-flex align-items-center justify-content-between",
+              {
+                active: step === i,
+                completed: i < step && !skipped?.has(i),
+                disabled: !enabled,
+              }
+            )}
+            key={i}
+          >
+            <a
+              key={i}
+              role="button"
+              className={clsx("nav-link")}
```

Comment:
dont think we need clsx here. you should just be able to do `classname="nav-link"`